core/src/test/kotlin/io/specmatic/core/HttpStubTests.kt

```diff
@@ -843,18 +876,50 @@ Scenario: JSON API to get account details with fact check
             Error from contract ${openApiFile.canonicalPath}
             In scenario "PATCH /creators/(creatorId:number)/pets/(petId:number). Response: pet response"
             API: PATCH /creators/(creatorId:number)/pets/(petId:number) -> 201
-            >> REQUEST.PARAMETERS.PATH.creatorId
-            Expected number, actual was "ABC"
-            >> REQUEST.PARAMETERS.PATH.petId
-            Expected number, actual was "DEF"
-            >> REQUEST.PARAMETERS.QUERY.creatorId
-            Expected number, actual was "ABC"
-            >> REQUEST.PARAMETERS.HEADER.PET-ID
-            Expected number, actual was "DEF"
-            >> REQUEST.BODY.creatorId
-            Expected number, actual was "ABC"
-            >> RESPONSE.BODY.petId
-            Expected number, actual was "DEF"
+            ${
```

Comment:
The errors that show up seem confusing. Seems like 3 errors, but it's actually the same mismatch causing two errors..?